lib/l10n/app_en.arb

```diff
@@ -302,7 +305,22 @@
   "pageSearchRideButtonSearch": "Search",
 
   "pageSearchSuggestionsTitle": "Search Suggestions",
-  "pageSearchSuggestionsButtonFilter": "Filter",
+  "pageSearchSuggestionsEmpty": "Nothing to see here",
+  "pageSearchSuggestionsRelaxRestrictions": "Relax your search restrictions.",
+  "pageSearchSuggestionsNoResults": "We're sorry, we couldn't find a ride for this route.",
+  "pageSearchSuggestionsTooltipFilter": "Filter",
+
+  "searchSuggestionsFilterFeaturesSelectTooltip": "Select",
+  "searchSuggestionsFilterFeatures": "Features",
+  "searchSuggestionsFilterFeaturesDeselectTooltip": "Deselect",
+  "searchSuggestionsFilterDeviation": "Maximum deviation from selected time",
+  "searchSuggestionsFilterMinimumRating": "Minimum rating",
+  "searchSuggestionsFilterDeviationHours": "hours",
+  "searchSuggestionsFilterSorting": "Sorting",
+  "searchSuggestionsFilterResetToDefault": "Reset to default",
+  "searchSuggestionsSortingRelevance": "Relevance",
+  "searchSuggestionsSortingTime": "Time",
```

Comment:
I would call "Time" "Traveltime". To me its not intuitive what Time mean (Starttime, Endtime, Traveltime)